### ISSUE TITLE
Deb package from PPA not installable on Debian 

### ISSUE THREAD

**Issue by quentinhaenn** [sosy-lab/benchexec#880]
The debian package is not installing because of a compression tool not supported by dpkg-deb :`.tar.zst` 

This has already been treated for a previous version here : #818 

![image](https://user-images.githubusercontent.com/83473151/202686336-e27cf464-1e6b-4ccd-864b-471618265e79.png)

**Comment by quentinhaenn** [comment]
Yes I do use your PPA, in fact I followed the instruction mentioned [here](https://github.com/sosy-lab/benchexec/blob/main/doc/INSTALL.md). 
This lead me to use those commands :
```bash 
sudo add-apt-repository ppa:sosy-lab/benchmarking
sudo apt install benchexec
̀̀```
And then I got the error mentioned. I also tried with downloading the `.deb` archive as its also mentioned in your install process, but got the same exact error. 

Unfortunately I cannot say if it worked for 3.11, but I managed to install 3.13 correctly somehow.


I cannot help you either to make Launchpad build your package so it works on Debian, sorry.
I

**Comment by PhilippWendler** [comment]
You downloaded [this file](https://github.com/sosy-lab/benchexec/releases/download/3.14/benchexec_3.14-1_all.deb) and it produced a compression error? That would be highly surprising, as I have confirmed manually that it uses the XZ compression, which is supported by Debian, and not ZST. And in fact I can install it just fine on Debian (just tried it out on Debian 11).

On the other hand BenchExec 3.13 from the PPA did use the unsupported ZST. Maybe you have mixed something up?

A final resolution for this will be if [this Debian bug](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=892664) is fixed, or if we find a way to publish PPAs specifically for Debian. Until then, I guess Debian users have to install the BenchExec package manually.

**Comment by quentinhaenn** [comment]
I tried again with your file linked, and it worked, I can't really explain how I have mixed it up, but I definitely downloaded the exact same file and then produced the error,  as I saved the terminal process and the error to check each step.  

Although it produced the error using the PPA for 3.14 and 3.13, I managed to install it manually too.

Thanks for sharing this Debian bug and for answering me, I guess this should be closed now. 
Have a nice day !
